baseline/OSPS-BR.yaml

```diff
@@ -349,6 +378,15 @@ controls:
         identifiers:
           - 1.1
           - 1.2
+      - reference-id: 800-161
+        identifiers: 
+          - AC-4
+          - CM-2
+          - CM-7(4)
+          - CM-7(5)
+          - RA-5
```

Comment:
RA-5 is about vulnerability monitoring/scanning. While a good idea, I don't read this Baseline control to include that. I suggest we drop this entry from the mapping.